### PR TITLE
Fixed dashboard designer and banner related issues

### PR DESCRIPTION
**Pull request by lasanthaS**

### CODE REVIEW

File: apps/portal/controllers/create-gadget.jag [hunk 1]
```diff
@@ -4,13 +4,19 @@
         sendLogin();
         return;
     }
+    
+    var configs = require('/configs/designer.json');
+    
+    if (!configs.isCreateGadgetEnable) {
+        response.sendError(404, 'not found');
```
Comment (TanyaM) [inline apps/portal/controllers/create-gadget.jag]: Make error message meaningful

File: apps/portal/js/jquery.gridster.override.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+(function() {
```
Comment (TanyaM) [inline apps/portal/js/jquery.gridster.override.js]: License?

File: apps/portal/controllers/create-gadget.jag (hunk 1 above)
Comment (lasanthaS) [inline apps/portal/controllers/create-gadget.jag]: Fixed in the PR https://github.com/wso2/carbon-dashboards/pull/174

File: apps/portal/js/jquery.gridster.override.js (hunk 2 above)
Comment (lasanthaS) [inline apps/portal/js/jquery.gridster.override.js]: This file is no longer avaiilable

**Pull request merged**